### ISSUE TITLE
Google API key

### ISSUE THREAD

**Issue by bitops** [taf2/speech2text#19]
Hello, it looks like an API key is required. Is there a way to provide the key value? I assume I have to sign up for the key myself, etc.

**Comment by philc** [comment]
Yes, I get this as well. Running speech2text stumbles on this 500 error page from google:

`Your client has issued a malformed or illegal request. Missing parameter: key`

**Comment by hendrikstier** [comment]
Any news on this? Where can I provide my key? Or is it simply not supported?

**Comment by taf2** [comment]
i think this is br0ken google changed their API.

**Comment by hendrikstier** [comment]
Too bad. Are there any plans to update the client to access the new API?

**Comment by taf2** [comment]
I'm pretty sure you need to become a chromium developer to get access to the keys and even then don't expect to get much usage out of those keys... unfortunately for whatever reason google doesn't appear to want to make money..

**Comment by hendrikstier** [comment]
It was no problem registering as a chromium dev and retrieving a key, but my first few attempts of including the key (either in HTTP headers or in the URL) did not succeed. A little bit of research also showed, that the API is aimed at a few words or sentences, and not to convert a complete speech, as it was my use case. Do you have a suggestion for a tool that covers this scenario?
